public/utils/helpers.js

```diff
@@ -160,7 +160,9 @@ export function initManageChannelsUrl(httpClient) {
     const relativePath = `/app/${
       getUseUpdatedUx() ? 'channels' : 'notifications-dashboards'
     }#/channels`;
-    MANAGE_CHANNELS_URL = httpClient.basePath.prepend(relativePath);
+    MANAGE_CHANNELS_URL = httpClient.basePath.prepend(relativePath, {
```

Comment:
Let's verify this is only used in one place and that the functionality doesn't break when new home page is disabled

Reply:
This is used in the create and edit monitor workflow as the link for the `manage channels` button but it appends the correct base path for when home page is both enabled and disabled.